src/main/java/pl/edu/agh/ki/io/models/User.java

```diff
@@ -30,9 +27,13 @@
     @Column(name = "login", unique = true, nullable = false)
     private String login;
 
-    @Column(name = "password", nullable = false)
+    @Column(name = "password", nullable = true)
```

Comment:
`nullable = true` jest domyslne wiec mozna to wywalic

Reply:
zostawiłem żeby pokazać jaka zmiana zaszła, ale już to wywalę